demo/simple-overlay.html

```diff
@@ -12,19 +12,16 @@
 <link rel="import" href="../iron-overlay-behavior.html">
 
 <dom-module id="simple-overlay">
-
-  <style>
-
-    :host {
-      background: white;
-      color: black;
-      padding: 24px;
-      box-shadow: rgba(0, 0, 0, 0.24) -2.3408942051048403px 5.524510324047423px 12.090680100755666px 0px, rgba(0, 0, 0, 0.12) 0px 0px 12px 0px;
-    }
-
-  </style>
-
   <template>
+    <style>
+      :host {
+        background: white;
+        color: black;
+        padding: 24px;
+        box-shadow: rgba(0, 0, 0, 0.24) -2.3408942051048403px 5.524510324047423px 12.090680100755666px 0px, rgba(0, 0, 0, 0.12) 0px 0px 12px 0px;
```

Comment:
lol, I can't take credit unfortunately, I found it like this when I started working on this repo https://github.com/PolymerElements/iron-overlay-behavior/commits/master/demo/simple-overlay.html